Dockerfile

```diff
@@ -10,7 +10,7 @@ RUN apt-get update && \
     localedef -i en_US -c -f UTF-8 -A /usr/share/locale/locale.alias en_US.UTF-8 && \
     apt-get install --no-install-recommends -y \
         python3-pip python3-setuptools python3-wheel python3-yaml python3-distro python3-jsonschema python3-newt \
-        gosu lsb-release file vim less procps tree tar bzip2 zstd pigz lz4 tmux libncurses-dev \
+        gosu lsb-release file vim less procps tree tar bzip2 zstd pigz lz4 unzip tmux libncurses-dev \
```

Comment:
when we have it here we do no longer need it in the "kas" container for yocto, final container of that Dockerfile